ament_cmake_gmock/ament_cmake_gmock-extras.cmake

```diff
@@ -36,6 +42,7 @@ macro(_ament_cmake_gmock_find_gmock)
         NO_DEFAULT_PATH NO_CMAKE_FIND_ROOT_PATH
       )
       find_file(_gmock_src_file "gmock.cc"
+        NAMES "gmock-gtest-all.cc"  # Alternative when using "fused" sources.
```

Comment:
Does this still find `gmock.cc`? According to the docs the single name and `NAMES` are mutually exclusive.